### DISCUSSION TITLE
Grid feed-in entity

### DISCUSSION THREAD

**Discussion by FelixIsReal** [weltenwort/home-assistant-rct-power-integration#372] [Q&A] [answered]
Hello everyone,

Today I integrated the RCT system into Homeassistant and looked at every possible entity that would look like it would show me how many kWh I have fed into the grid today. Not the home grid with battery but the grid that is connected to the electricity meter and for which I am paid for feeding in.

Can anyone tell me what the entity is called?

Sincerely yours

**Comment by gt026328** [comment]
I guess it is "  RCT ...Grid Energy Production Day"

**Reply by FelixIsReal** [reply, marked as answer]
I'm sorry. 
In that time I tried ,,grid Power", that's the right one, but thanks.
